src/test/suite/extension.test.ts

```diff
@@ -17,19 +17,31 @@ suite('completion', () => {
         cleanUpFiles();
     });
 
-    test('liam.', async () => {
+    test('liam.', async function () {
         const doc = await openTargetFile(path.join(simpleProgramPath, 'student.rb'));
-        const list = (await vscode.commands.executeCommand(
-            'vscode.executeCompletionItemProvider',
-            doc.uri,
-            new vscode.Position(13, 18),
-        )) as vscode.CompletionList;
+        const pos = new vscode.Position(13, 18);
+        const list = await retryUntil(
+            async () => {
+                const cmd = 'vscode.executeCompletionItemProvider';
+                const list = await vscode.commands.executeCommand<vscode.CompletionList>(cmd, doc.uri, pos);
+                if (
+                    list &&
+                    list.items.length > 0 &&
+                    list.items.some((item) => item.kind === vscode.CompletionItemKind.Method)
```

Comment:
[note] `list.items` sometimes include only `CompletionItemKind.Text` items. So, this check using `some()` is necessary.